service/component-samples/http-device-sample/pom.xml

```diff
@@ -132,6 +135,47 @@
               <includeScope>runtime</includeScope>
             </configuration>
           </execution>
+          <execution>
+            <id>copy-installed</id>
+            <phase>package</phase>
+            <goals>
+              <goal>copy</goal>
+            </goals>
+            <configuration>
+              <artifactItems>
+                <artifactItem>
+                  <groupId>${project.groupId}</groupId>
+                  <artifactId>${project.artifactId}</artifactId>
+                  <version>${project.version}</version>
+                  <type>${project.packaging}</type>
+                  <outputDirectory>${project-demo-directory}/${project.artifactId}</outputDirectory>
+                  <destFileName>${project.artifactId}.${project.packaging}</destFileName>
```

Comment:
Since we are copying JAR file and lib folder to demo directory, I would recommend to add the maven clean plugin to ensure that those items are removed during maven clean goal execution. 
For reference: https://github.com/secure-device-onboard/pri-fidoiot/blob/master/service/component-samples/rv/pom.xml Line 188 - 207.

Reply:
Done.